src/components/Provider/ProviderHeader/ProviderHeader.js

```diff
@@ -18,14 +19,25 @@ const ProviderHeader = () => {
   const storeSettings = useStoreSettingsSelector();
   const providerId = typeof window !== "undefined" ? location.pathname.split("/")[2] : "";
   const [links, setLinks] = useState([]);
+  const { emptySettingsAlert } = useContext(AppContext);
 
   useEffect(() => {
     const data = provider.isCopyTrading
       ? createTraderRoutes(providerId, provider)
       : createProviderRoutes(providerId, provider, storeSettings.selectedExchange);
+    if (!provider.isCopyTrading) {
+      data.links.some((item) => {
```

Comment:
It should be a simple map I think

Reply:
It can't be a map function because I am updating property for a single object in "links" array.

Reply:
right, then forEach. `some` is to know if an array element matches the passed function